site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Segmented Controls
+description: Segmented controls component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - segmented-control
+  - segmented-controls
+  - segmented-controls-footer
+---
+
+## Segmented Control
```

Comment:
If you like, we can revisit this in a follow up PR, but I think this component documentation should probably be for the styling. The documentation here seems to be primarily for the JavaScript/Web Component aspect of it, which should be separate from the styling (since they are de-coupled). I think the doc for the JS should be in a separate folder, maybe called JS or web-components.

Reply:
I will make sure this is my first follow up.